scripts.js

```diff
@@ -6,6 +6,19 @@ var options = {
     precedence: document.getElementById('option-precedence'),
 };
 
+var statistics = {
```

Comment:
Since we now have a lot of elements declared, could we put `mapElement`, `options`, and `statistics` under a single object called `elements`/`elem`/`e`/`page` or something to that effect? I think it could make the code a little cleaner.

Reply:
If we split `scripts.js` into multiple files, this might not be a good choice. Although we could still do something like
```js
elements.statistics = {
    // ...
};
```
in a hypothetical `statistics` file.